app/js/views/services-list-item.jsx

```diff
@@ -8,10 +8,13 @@ export default class ServicesListItem extends React.Component {
       available: false,
       on: true,
       locked: true,
+      motionDetected: false,
```

Comment:
More than agreed here. We need something similar to `service.jsx` or even share some logic between both views.